packages/mdc-bottom-line/README.md

```diff
@@ -33,24 +33,24 @@ The bottom line indicates where to enter text, displayed below the label. When a
 <div class="mdc-text-field">
   <input type="text" id="my-text-field-id" class="mdc-text-field__input">
   <label class="mdc-text-field__label" for="my-text-field-id">Hint text</label>
-  <div class="mdc-text-field__bottom-line"></div>
+  <div class="mdc-bottom-line"></div>
 </div>
 ```
 
 ### CSS Classes
 
 CSS Class | Description
 --- | ---
-`mdc-text-field-bottom-line` | Mandatory
-`mdc-text-field-bottom-line--active` | Styles the bottom line as an active bottom line
+`mdc-bottom-line` | Mandatory
+`mdc-bottom-line--active` | Styles the bottom line as an active bottom line
 
-### `MDCTextFieldBottomLine`
+### `MDCBottomLine`
 
-##### `MDCTextFieldBottomLine.foundation`
```

Comment:
I think, since we are making this a public component, we should not make the foundation a field of `MDCBottomLine` anymore. I think we should create proxy methods on `MDCBottomLine` which proxy to `MDCBottomLineFoundation`'s `activate`, `deactivate` and `setTransformOrigin`

When it was internal only to text-field, it was easier not to create proxy methods. But I think with this becoming a public component we should protect access to the foundation class.

Of course, you'll also have to change the adapter for Text Field. I think Select has a good example of this here:
https://github.com/material-components/material-components-web/blob/master/packages/mdc-select/index.js#L112

It shows how the Select foundation interacts with a Menu through the Select adapter. I think we need to add methods like `activateBottomLine` to the Text Field adapter.